rpmautospec/specparser.py

```diff
@@ -0,0 +1,98 @@
+import copy
+import sys
+from rpmautospec_core import AUTORELEASE_MACRO
+
+try:
+    from norpm.macrofile import system_macro_registry
+    from norpm.specfile import specfile_expand, specfile_expand_string
+except ImportError:
+    pass
+
+from .compat import rpm
+
+AUTORELEASE_DEFINITION = "E%{?-e*}_S%{?-s*}_P%{?-p:1}%{!?-p:0}_B%{?-b*}"
+
+class SpecParser:
```

Comment:
I don't really understand why you're implementing both the old and new parsers here. If the NoRPM approach works, we do we need to retain both?

Reply:
Trying to explain above in https://github.com/fedora-infra/rpmautospec/pull/319#discussion_r2397273415